tests/helpers/broadcasts.ts

```diff
@@ -0,0 +1,11 @@
+import type { Sablier } from "@src/types";
+
+/**
+ * Determines if a release uses the same broadcast directory for ZK and non-ZK chains.
+ */
```

Comment:
Let's please expand the definition to explain why the new ZK deployments are unified.